src/components/CodeViewer/SourceCodeViewer.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles/index";
+import AceEditor from "react-ace";
+import PropTypes from "prop-types";
+import Paper from "@material-ui/core/Paper";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpandMoreIconRounded from "@material-ui/icons/ExpandMoreRounded";
+import Typography from "@material-ui/core/Typography";
+import "brace/theme/twilight";
+import "brace/mode/javascript";
+
+const styles = theme => ({
+    aceEditor: {
+        width: "100% !important",
+        position: "relative",
+        marginBottom: theme.spacing(1)
+    },
+    expansionPanelContainer: {
+        margin: "1rem 0"
+    }
+});
+
+function SourceCodeViewer(props) {
+    const { classes, code } = props;
```

Comment:
You can use `useStyles` hook for this. It will save you a prop.

Reply:
Good tip, I updated to `useStyles`